processing/thermal.py

```diff
@@ -544,8 +622,22 @@ def load(cls, raw_track):
             start_s=raw_track.get("start_s"),
             end_s=raw_track.get("end_s"),
             score=raw_track.get("tracking_score"),
+            thumbnail_info=raw_track.get("thumbnail"),
         )
 
+    def post_data(self):
+        data = {
+            "id": self.id,
```

Comment:
Should probably only add id if it has one, as you use post_data in add_track (which I presume it doesn't have yet)